src/Core/Framework/Demodata/Command/DemodataCommand.php

```diff
@@ -87,9 +87,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->ensureAllDependenciesArePresent();
 
         if ($this->kernelEnv !== 'prod') {
-            $output->writeln('Demo data command should only be used in production environment. You can provide the environment as follow `APP_ENV=prod framework:demodata`');
+            $output->writeln('Demo data command should only be used in production environment. You can provide the environment as follows `APP_ENV=prod bin/console framework:demodata`');
 
-            return self::SUCCESS;
+            return self::INVALID;
```

Comment:
I changed this for semantic reasons. Seams weird to return a SUCCES while throwing an error